pkg/scheduler/scheduler.go

```diff
@@ -466,7 +466,9 @@ func (sched *Scheduler) scheduleOne() {
 					" No preemption is performed.")
 			} else {
 				preemptionStartTime := time.Now()
-				sched.preempt(pod, fitError)
+				if err := sched.preempt(pod, fitError); err != nil {
+					klog.Warningf("%v/%v preempting lower priority pods fail: %v", pod.Namespace, pod.Name, err)
```

Comment:
The `preempt` function already logs error messages. We don't need to double log here.